src/web/routes/application/flow-control/state-machine/operators.test.js

```diff
@@ -0,0 +1,99 @@
+const test = require('tape')
+const { JOURNEYS_KEY, NEXT_ALLOWED_PATH_KEY, STATE_KEY } = require('./keys')
+const { setNextAllowedPathInSession, setStateInSession, getNextAllowedPathFromSession, getStateFromSession } = require('./operators')
+
+const journey = { name: 'report-a-change' }
```

Comment:
Can we make this upper case so its more obvious in the tests that it a constant in the file? Too me a bit to figure out where `journey` was coming from in the tests.

Reply:
Yep.